vortex/Cargo.toml

```diff
@@ -67,6 +67,11 @@ python = ["vortex-error/python"]
 tokio = ["vortex-file/tokio", "vortex-scan/tokio"]
 zstd = ["dep:vortex-zstd", "vortex-file/zstd", "vortex-layout/zstd"]
 pretty = ["vortex-array/table-display"]
+layout_names = [
+    "vortex-file/layout_names",
+    "vortex-layout/layout_names",
+    "vortex-scan/layout_names",
+]
```

Comment:
Should it be part of the "pretty" feature? maybe it should be re-named to "verbose" or something similar